eventdisplay/FairEventManagerEditor.cxx

```diff
@@ -229,7 +229,6 @@ void FairEventManagerEditor::StartAnimation()
             FairTask* pMainTask = ana->GetMainTask();
             TList* taskList = pMainTask->GetListOfTasks();
 
-            Int_t ntask = ana->GetNTasks();
```

Comment:
This change looks like it is meant to belong to some other commit?

Reply:
You're right. I'll move it into its own commit.

Reply:
done.